src/main/java/seedu/address/model/lesson/Location.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.lesson;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.contact.Name;
+
+public class Location {
+    public final String location;
+
+    public Location(String location) {
+        requireNonNull(location);
+        this.location = location;
+    }
+
+    @Override
+    public String toString() {
+        return location;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+            || (other instanceof Name // instanceof handles nulls
```

Comment:
Can I check why we are checking if other is instanceof Name? We should only check if other is also a Location right

Reply:
My bad😢